var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -216,7 +217,8 @@ def edit(self, spec, prefix):
         if '+mpi' in self.spec:
             cppflags.extend([
                 '-D__parallel',
-                '-D__SCALAPACK'
+                '-D__SCALAPACK',
+                '-D__SCALAPACK2',
```

Comment:
I wish people stop using reserved names for their macros :sob:

Reply:
I call that `hysterically grown`